mappings/net/minecraft/client/particle/ParticleSpell.mapping

```diff
@@ -32,7 +32,7 @@ CLASS none/bnu
 			ARG 6 velocityY
 			ARG 7 velocityZ
 			ARG 8 extra
-	CLASS none/bnu$d
+	CLASS none/bnu$d Factory
```

Comment:
Maybe `FactorySwirl`? `FactoryNormal`? (assuming this is the normal "swirl" potion particle)